other/dlc.py

```diff
@@ -117,6 +145,82 @@ def filter_list_by_date(data, token):
     return ret
 
 
+def todays_g4_event_filename(today, gamecd, dlc_path):
+    event_filename = "null"
```

Comment:
It might save some logic if you returned 'None' when the event evaluation below failed